hardware/scripts/board.js

```diff
@@ -1,35 +1,53 @@
 'use strict';
 
-function createElement(id, className) {
-  let element = document.createElement('div');
-  element.id = id;
-  element.className = className;
-  return element;
-}
-
-function buildBoard() {
-  let board = createElement('board', 'board');
+{
```

Comment:
I've put it inside a new scope so that I can require the Hardware object. (with "const" and "let" we don't need a IIFE anymore :) )